### PR TITLE
www-misc/monitorix: Update for GLEP 81

### PR DESCRIPTION
**Pull request by Dwosky**

### CODE REVIEW

File: acct-user/monitorix/monitorix-0.ebuild [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="Monitorix system tool user"
+ACCT_USER_ID=401
+ACCT_USER_HOME=/var/lib/monitorix
```
Comment (juippis) [inline acct-user/monitorix/monitorix-0.ebuild:10]: Is this needed?
http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml#the-home-directory

File: acct-user/monitorix/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<pkgmetadata>
+	<maintainer type="person">
+		<email>dwosky@pm.me</email>
+		<name>Pedro Arizmendi</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+</pkgmetadata>
```
Comment (juippis) [inline acct-user/monitorix/metadata.xml:12]: Github shows these errors in your files, I'm not really sure what they mean, but probably something with your editor. Could you check?

Comment (thesamesam) [reply acct-user/monitorix/metadata.xml:12]: I think it's the missing newline at the end.

Comment (juippis) [reply acct-user/monitorix/metadata.xml:12]: I mean, yes, that much was obvious, but I don't remember how to fix these. I had the same error when I used emacs. So I'd say it's something in your editor's configuration.

Comment (Dwosky) [reply acct-user/monitorix/metadata.xml]: Not sure why it was like that, I've checked both user and group file and should be corrected.

File: acct-user/monitorix/monitorix-0.ebuild (hunk 1 above)
Comment (Dwosky) [reply acct-user/monitorix/monitorix-0.ebuild]: Didn't saw that article when checking for GLEP 81. In fact it isn't really needed, so I've removed it.

**Pull request closed without merging**